Cython/Utility/TypeConversion.c

```diff
@@ -710,6 +710,13 @@ static CYTHON_INLINE PyObject* {{TO_PY_FUNCTION}}({{TYPE}} value, Py_ssize_t wid
 
 // NOTE: inlining because most arguments are constant, which collapses lots of code below
 
+// GCC diagnostic pragmas were introduced in GCC 4.6
+#ifdef __GNUC__
+#define GCC_VERSION (__GNUC__ * 10000 + __GNUC_MINOR__ * 100 + __GNUC_PATCHLEVEL__)
```

Comment:
Is there a more central location/file where `GCC_VERSION` should be defined?

Reply:
Just check major/minor separately, like it's done in other places.

Reply:
Thanks for the feedback.  I've made the requested change.